src/plugins/implementations/highlight.ts

```diff
@@ -23,6 +23,8 @@ export interface HighlightPluginOptions extends PluginOption {
 
 export default class HighlightPlugin extends CommitPlugin {
     private masks: RegExp[] = [];
+    private isEscaped = false;
+    private replacements: Map<number, string> = new Map();
```

Comment:
These variables cannot be plugin fields since they are local to each commit that the plugin processes. By making them plugin fields, you create a potential error when processing them.